Song_sort_Genre.php

```diff
@@ -60,7 +94,7 @@
 					
 				while($output = mysqli_fetch_array($result))
 					{
-			?>
+				?>
 				<Space></Space>
 				<h3> <p class="white"><?php echo $output['Title'];?></p></h3>
```

Comment:
21/9: This will make the data will display right underneath the title that it belong to.